docs/examples/simple_examples.rst

```diff
@@ -0,0 +1,174 @@
+line of sight, and the dynamic spectrum of the observation will look rather
+plain. In the secondary spectrum, there will be a single non-zero point at the
+origin.
+
+.. plot::
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [0.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs, mu_eff=0.*u.mas/u.yr, screens=False,
+                 velocity=False)
+
+A scattering screen
+===================
+
+Now introduce a scattering screen somewhere in the space between the pulsar and
+Earth. This screen, a one-dimensional lens, will give rise to point-like
+scattering images of the pulsar. As a first step, we consider the effect of a
+single scattering image. In addition to the direct line of sight, there will
+now be a scattered beam via which radiation from the pulsar reaches Earth.
+
+The radiation going via the
+scattered beam takes a longer path to Earth, and the resulting delay in arrival
+times between the two beams gives rise to an interference pattern in the
+dynamic spectrum. The interference pattern is stable in time (i.e., the dynamic
+spectrum is not really dynamic yet).
+In the secondary spectrum, two points show up along the vertical axis,
+corresponding to the relative geometric delay between the direct line of sight
+and the scattered beam.
+The reason there are two points, rather than one, is that the data (the
+interference pattern) do not distinguish which of the two images arrives first.
+
+.. plot::
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [0., 2.] << u.mas
+    magnifs = gen_magnifs(theta)
+
+    example_figs(theta, magnifs, mu_eff=0.*u.mas/u.yr, velocity=False)
+
+The effect of relative motion
+=============================
+
+Until now we have assumed there is no relative motion between the pulsar,
+Earth, and the scattering screen (i.e., the interstellar medium).
+While in reality all these components will have their own space motion,
+they can all be included by a single effective velocity with which
+the line of sight moves with respect to the scattering screen.
+
+This relative motion (represented in the schematic below by the arrow next to
+the pulsar) causes the delay between the two beams to change with time.
+As a result, the interference pattern resulting from their interaction shifts
+with time, seen as diagonal bands in the dynamic spectrum.
+In the secondary spectrum, the two points that were previously on the
+vertical axis have shifted to the differential Doppler shift corresponding to
+the effective velocity of the system.
+
+.. plot::
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from example_figure import gen_magnifs, example_figs
+
+    theta = [0., 2.] << u.mas
+    magnifs = gen_magnifs(theta)
```

Comment:
Here, using exactly the same magnifications as in the previous example would be helpful.